source/developer/developer-setup.md

```diff
@@ -82,6 +82,10 @@ Any issues? Please let us know on our forums at: http://forum.mattermost.org
 
 ### Archlinux ###
 
+_TIP_: There is a package in the [Arch User Repository](https://aur.archlinux.org/) called [Mattermost](https://aur.archlinux.org/packages/mattermost/). Follow the [AUR instructions](https://wiki.archlinux.org/index.php/Arch_User_Repository) to build and install. 
```

Comment:
Thanks for the addition! Seems helpful. 

Propose being explicit this is an unofficial package and sharing how to find support, e.g.: 

```
There is an unofficial package in the [Arch User Repository](https://aur.archlinux.org/) maintained by [mtorromeo](https://github.com/mtorromeo/), called [Mattermost](https://aur.archlinux.org/packages/mattermost/). Follow the [AUR instructions](https://wiki.archlinux.org/index.php/Arch_User_Repository) to build and install. For sharing feedback, troubleshooting and bug reports, please leave a comment at: https://aur.archlinux.org/packages/mattermost/
```

Alternatively we could direct people to https://pre-release.mattermost.com/core/channels/installers-and-images if @mtorromeo might be interested in fielding questions there. 

Just some ideas.